### DISCUSSION TITLE
postinstalling git failing on macOS Monterey 12.6.3

### DISCUSSION THREAD

**Discussion by burlinif** [Homebrew/discussions#4268] [Getting started]
### Output of `brew config`

```shell
brew config
HOMEBREW_VERSION: 4.0.3
ORIGIN: https://github.com/Homebrew/brew
HEAD: ba6d87eed9ce31f42fa2b33a4e6067d9cf17f13f
Last commit: 4 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 426a101514ecfdc5a146464a99ab4e1cb61074ee
Core tap last commit: 2 years, 2 months ago
Core tap branch: master
Core tap JSON: 24 Feb 12:19 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 14.0.0 build 1400
Git: 2.39.2 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.3-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
brew doctor
==> Downloading https://formulae.brew.sh/api/formula.json

Your system is ready to brew.
```

### Description of issue

I am trying to install Git using homebrew. I have tried reinstalling it multiple times, however the post-install step never completes successfully.

francescaburlini$ brew reinstall git
==> Fetching git
==> Downloading https://ghcr.io/v2/homebrew/core/git/manifests/2.39.2-1
Already downloaded: /Users/francescaburlini/Library/Caches/Homebrew/downloads/bfede1d08ef39a607d8898c0738a332be30aded1421abc780d2ed50d8904cd1d--git-2.39.2-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/git/blobs/sha256:77125b873d51672df91f127226
Already downloaded: /Users/francescaburlini/Library/Caches/Homebrew/downloads/44eeddf488ccb3c0c0cf719e50e17a76160433ffecf61cb71b960b5ac7e9f190--git--2.39.2.monterey.bottle.1.tar.gz
==> Reinstalling git 
==> Pouring git--2.39.2.monterey.bottle.1.tar.gz
Warning: The post-install step did not complete successfully
You can try again using:
  brew postinstall git
==> Caveats
The Tcl/Tk GUIs (e.g. gitk, git-gui) are now in the `git-gui` formula.
Subversion interoperability (git-svn) is now in the `git-svn` formula.

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> Summary
🍺  /usr/local/Cellar/git/2.39.2: 1,627 files, 47.5MB
==> Running `brew cleanup git`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

**Comment by SMillerDev** [comment]
What does `brew postinstall --debug git` say?

**Reply by burlinif** [reply]
$ brew postinstall --debug git
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaAPILoader): loading git from API
==> Postinstalling git
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromPathLoader): loading /usr/local/opt/git/.brew/git.rb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromPathLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git.rb
Warning: The post-install step did not complete successfully
You can try again using:
  brew postinstall git
==> An exception occurred within a child process:
  FormulaUnreadableError: git: Invalid bottle tag symbol

/usr/local/Homebrew/Library/Homebrew/utils/bottles.rb:143:in `from_symbol'
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:600:in `sha256'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git.rb:16:in `block in <class:Git>'
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:128:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:128:in `bottle'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2958:in `bottle'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git.rb:15:in `<class:Git>'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git.rb:1:in `block in load_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:93:in `module_eval'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:93:in `block in load_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:105:in `load_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:125:in `load_formula_from_path'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:387:in `load_file'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:377:in `klass'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:372:in `get_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:629:in `factory'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:295:in `resolve'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:180:in `resolve_formula'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:100:in `load_formula_or_cask'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:57:in `block in to_formulae_and_casks'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:56:in `each'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:56:in `flat_map'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:56:in `to_formulae_and_casks'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:186:in `to_resolved_formulae'
/usr/local/Homebrew/Library/Homebrew/postinstall.rb:22:in `<main>'
